build/docker/tools/tke-installer/Dockerfile

```diff
@@ -14,14 +14,15 @@
 # WARRANTIES OF ANY KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations under the License.
 
-FROM alpine:3.10
+FROM docker:dind
```

Comment:
replace tag to 20.10.8-dind
dind tag will always pull latest image

Reply:
Done.